### PR TITLE
fix test_word2vec bug

### PR DESCRIPTION
**Pull request by fc500110**
compare floats number for almost equality

### CODE REVIEW

File: python/paddle/fluid/tests/book/test_word2vec.py [hunk 1]
```diff
@@ -220,9 +220,7 @@ def to_infer_tensor(lod_tensor):
         np_data = np.array(results[0])
         infer_out = infer_outputs[0].data.float_data()
         for a, b in zip(np_data[0], infer_out):
-            g_a = float("{:.6g}".format(a))
-            g_b = float("{:.6g}".format(b))
-            assert g_a == g_b
+            assert np.isclose(g_a, g_b)
```
Comment (Superjomn) [inline python/paddle/fluid/tests/book/test_word2vec.py]: isclose, set a retio 1e-4 ?

Comment (fc500110) [reply python/paddle/fluid/tests/book/test_word2vec.py]: The default relative tolerance parameter is 1e-5, this is the float's accuracy.

**Review by Superjomn — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/book/test_word2vec.py (hunk 1 above)
Comment (Superjomn) [reply python/paddle/fluid/tests/book/test_word2vec.py]: add some debug information if this assertion fail

such as 

```python
assert np.isclose(...), "g_a: %f, g_b: f" % (g_a, g_b)
```

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**